release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1097,6 +1097,91 @@ This script removes unauthenticated subjects from the following cluster role bin
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1821771[*BZ#1821771*])
 
+// TODO: This known issue should be removed when RHCOS images begin using RHEL 9. RHEL 9 prevents this issue from occurring.
+* Intermittently, an IBM Cloud VPC cluster might fail to install because some worker machines do not start. Rather, these worker machines remain in the `Provisioned` phase. There is a workaround for this issue. From the host where you performed the initial installation, delete the failed machines and run the installation program again. (link:https://issues.redhat.com/browse/OCPBUGS-1327[*OCPBUGS#1327*])
++
+. Verify that the status of the internal application load balancer (ALB) for the master API server is `active`.
+.. Identify the cluster's infrastructure ID by running the following command:
++
+[source,terminal]
+----
+$ oc get infrastructure/cluster -ojson | jq -r '.status.infrastructureName'
+----
+.. Log into the IBM Cloud account for your cluster and target the correct region for your cluster.
+.. Verify that the internal ALB status is `active` by running the following command:
++
+[source,terminal]
+----
+$ ibmcloud is lb <cluster_ID>-kubernetes-api-private  --output json | jq -r '.provisioning_status'
+----
+. Optional: Verify that the traffic through the internal ALB is reaching the Machine Config Server (MCS) without failures:
+.. Identify the URI of the internal ALB by running the following command:
++
+[source,terminal]
+----
+$ oc get infrastructure/cluster -ojson | jq -r '.status.apiServerInternalURI'
+----
++
+.. Verify that the traffic is reaching the MCS by running the following command from a new machine (IBM Cloud Virtual Server Instance). This machine must be provisioned on the same subnet as one of the failed machines, and have the same security policies applied to it:
++
+----
+$ curl --max-time 5 --connect-timeout 5 --retry 10 https://api-int.<cluster_id>.<domain_name>:22623/healthz
+----
```

Comment:
I fail to verify this step now, and what is expected output ?

Reply:
@jeffnowicki if need much changed to check the traffic, still need add to the know issue ? how about add to the 
https://github.com/openshift/installer/tree/master/docs/user 
and simplify the know issue doc ?